src/components/shared/DotsMenu.js

```diff
@@ -18,6 +23,12 @@ export default class DotsMenu extends Component {
     this.setState({ anchorEl: null });
   };
 
+  handleShare = () => {
```

Comment:
please make dots menu work like in vanilla app (see https://github.com/ihor4568/spotty-vanilla/blob/master/src/components/MediaPlayer/MediaPlayer.js)